### PR TITLE
Update scenario source lisneter to work with http

### PR DESCRIPTION
**Pull request by MBMIII**

### CODE REVIEW

File: src/main/java/executor/service/controllers/ScenarioSourceController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
```
Comment (artemkedyk) [inline src/main/java/executor/service/controllers/ScenarioSourceController.java:15]:
```suggestion
@RequestMapping("/api/scenario")
```

File: src/main/java/executor/service/controllers/ScenarioSourceController.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+public class ScenarioSourceController {
+
+    private ScenarioSourceListener scenarioSourceListener;
+
+    public ScenarioSourceController(ScenarioSourceListener scenarioSourceListener) {
+        this.scenarioSourceListener = scenarioSourceListener;
+    }
+
+    @PostMapping("/scenario")
```
Comment (artemkedyk) [inline src/main/java/executor/service/controllers/ScenarioSourceController.java:24]:
```suggestion
    @PostMapping()
```

File: src/main/java/executor/service/controllers/ScenarioSourceController.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    public ResponseEntity<Integer> addScenario(@RequestBody Scenario scenario) {
+        ((ScenarioSourceListenerImpl) scenarioSourceListener).setScenario(scenario);
+        scenarioSourceListener.execute();
+        return ResponseEntity.ok(200);
+    }
+
+    @PostMapping("/scenario_list")
```
Comment (artemkedyk) [inline src/main/java/executor/service/controllers/ScenarioSourceController.java:31]:
```suggestion
    @PostMapping("/list")
```

**Review by artemkedyk — CHANGES_REQUESTED** [review]

**Comment by artemkedyk** [comment]
let's use this endpoint names

**Comment by MBMIII** [comment]
@artemkedyk done

**Review by artemkedyk — APPROVED** [review]

**Pull request merged**